### DISCUSSION TITLE
Is it planned to allow for swapping out the source in a player?

### DISCUSSION THREAD

**Discussion by wplit** [vidstack/player#818] [General]
I'm thinking in terms of adding a playlist.

For example if after a Youtube Video ends, then a Vimeo video starts to play.  Is there an intention to have something in the API to do this?

Or will this be a case of manually destroy and initialise a new player for the new video?

**Comment by mihar-22** [comment]
Hey @wplit!

Source and provider switching is automatically handled out of the box.

**Comment by wplit** [comment]
Ah man, that's awesome. Only just familiarising myself with it and must have missed that part.

Congrats on how far this has come, Vidstack looks great from what I've seen.

**Reply by mihar-22** [reply]
Thanks heaps! We're nearly at 1.0, so close now!
